docs/source/get_involved/governance.rst

```diff
@@ -320,6 +320,45 @@ For more details about our meetings, please go to our `community-council reposit
 
 To contact the CC directly, please send an email to info@sktime.org.
 
+.. _cc-observers:
+
+CC observers
+~~~~~~~~~~
+
+CC (community council) observers are core developers with additional rights and
+responsibilities. Current CC observers are listed in the `community-council
+observers <https://www.sktime.org/en/stable/about/team.html>`__ .
+
+.. _rights-and-responsibilities-4:
+
+Rights and responsibilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Observers have a full view of CC and participate in the decision making process
+during CC discussions to ensure that more members of the community have direct
+input on CC decisions.
+
+Eligibility
+^^^^^^^^^^^
+Only core developers are eligible for appointment as CC observers.
+Non-core-developers can be nominated, but this must be accompanied
+by a nomination for core developer, and a core developer appointment vote
+concurrent with the 5 day discussion period (see below).
+
+.. _appointment-4:
+
+Appointment
+^^^^^^^^^^^
+Membership of the CC observers is by nomination by a core developer and a vote by
```

Comment:
this is copy-pasting the wrong paragraph, no?

Approval should be by CC, not CoC committee.
Other than that, makes sense as a mechanism.

I would actually tend to reduce the majorities required - since the idea is input and transparency rather than actual decision making, my intuition in mechanism design would tell me that we should reduce the barrier more. Perhaps to one third of all cast votes?

Reply:
(so it can be a minority!)